lib/widgets/zendesk_chat_widget.dart

```diff
@@ -1,16 +1,43 @@
 import 'package:flutter/material.dart';
 import 'package:zendesk_chat/models/chat_settings_model.dart';
+import 'package:zendesk_chat/models/chat_style.dart';
 import 'package:zendesk_flutter_plugin/chat_models.dart';
 import 'package:zendesk_flutter_plugin/zendesk_flutter_plugin.dart';
 
+
+
+class _DefaultStyle extends ChatStyle {
+  final Color mainColor = Colors.blue;
+  final Color visitorColor = Colors.green;
+  final TextStyle nameTextStyle = const TextStyle(
+    fontWeight: FontWeight.bold,
+    color: Colors.white,
+  );
+
+  final TextStyle messageTextStyle = const TextStyle(
+    color: Colors.white,
+  );
+
+  const _DefaultStyle({
+    Color mainColor,
+    Color visitorColor,
+    TextStyle nameTextStyle,
+    TextStyle messageTextStyle,
+  });
+}
+
+const ChatStyle _defaultStyle = const _DefaultStyle();
```

Comment:
With this approach if I set just `mainColor` in the `chatStyle` all the defaultStyle will be replaced, it's ok. But in this case ChatStyle properties must be required and `assert` to not null.